### DISCUSSION TITLE
Modifying CSS on an imported plugin - (newsletter)

### DISCUSSION THREAD

**Discussion by clevesquedev** [GrapesJS/grapesjs#5248] [Q&A]
I am using the grapesjs newsletter plugin to help our application create email templates for our customers.
Since the newsletter imports in it's own blocks, is there a way to modify the CSS base of these blocks?

for instance the button block comes with a ".button" class which is

.button {
    padding-top: 30px;
    padding-right: 0px;
    padding-bottom: 30px;
    padding-left: 0px;
}

Is there a way to override these css values that the button block inherits from the plugin?

**Comment by artf** [comment]
If the block contains that CSS you can always [update its content programmatically](https://grapesjs.com/docs/modules/Blocks.html#programmatic-usage)
```js
// Get the Block
const block = bm.get('BLOCK-ID');
// Update the Block properties
block.set({
  content: '...new content...',
});
```
